README.md

```diff
@@ -107,7 +113,7 @@ configuration for the ECDSA signature is shown below in
 [print-your-cert-ca](#print-your-cert-ca). Also, I chose to have a very long
 expiry for both certificates since there is no security risk associated with
 leaking either of the private keys (since the private keys of both will be
-discarded on 21 May 2022 anyways).
+discarded on 23 March 2024 anyways).
```

Comment:
suggestion: I'm not sure if we should discard then!

We now have the ability to use the same root every kubecon and then swap out the intermediate. I think that's kinda cool!

Reply:
Agreed, discarding the private key would defeat the purpose of keeping the same root across kubecon